pegasus/sites.v3/hourofcode.com/public/index.haml

```diff
@@ -8,8 +8,6 @@ social:
 :ruby
   @header["title"] = hoc_s(:front_title, locals: {campaign_date: campaign_date('full-year')})
 
-  hoc_mode = DCDO.get("hoc_mode", CDO.default_hoc_mode)
```

Comment:
I think this line should be kept in since `hoc_mode` is referenced later

Reply:
Or does this file already have access to `hoc_mode` without declaring it here?

Reply:
Yep! It's on line 27 too (with a big comment) so I removed this one.